src/main/java/net/minestom/server/network/packet/server/play/data/DeathLocation.java

```diff
@@ -0,0 +1,11 @@
+package net.minestom.server.network.packet.server.play.data;
+
+import net.minestom.server.coordinate.Point;
+import org.jetbrains.annotations.NotNull;
+
+public record DeathLocation(
+        @NotNull String dimension,
+        @NotNull Point position
+) {
```

Comment:
Wrong formatting + empty line below

Reply:
A little confused as to what you mean. I assuming you'd prefer the formatting of the fields to be inline rather than line separated, but I'm confused as to what you mean by an empty line below. You'd prefer an empty record body to be inline too?

Reply:
I was referring to the empty line just below L10. Not that everything should be inlined